builtin/game/falling.lua

```diff
@@ -140,7 +140,17 @@ local function convert_to_falling_node(pos, node)
 	node.level = core.get_node_level(pos)
 	local meta = core.get_meta(pos)
 	local metatable = meta and meta:to_table() or {}
-
+	
```

Comment:
```suggestion

```

Reply:
Oops, this wasn't really exactly what I wanted to suggest. Do not remove the empty line.